frontend/wallet/src/render/components/Header.re

```diff
@@ -110,16 +113,16 @@ module SyncStatus = {
     /*     None; */
     /*   }); */
     switch ((result: SyncStatusQuery.response)) {
-    | Loading => <Alert kind=`Warning message="Connecting" />
-    | Error(_) => <Alert kind=`Danger message="Error" />
+    | Loading => <Alert kind=`Warning message="connecting" />
```

Comment:
See `Alert`. message is now passing down the react-intl id/message. Since I lowercase all ids, these props have to be lowercased.

Reply:
Makes sense. I guess the message prop is now messageID?

Reply:
defaultMessage is required for alert messages. But a messageID is optional to internationalize.